model-optimizer/mo/utils/cli_parser.py

```diff
@@ -1234,12 +1234,15 @@ def check_positive(value):
     return int_value
 
 
-def depersonalize(value: str):
+def depersonalize(value: str, key: str):
+    dir_keys = [
+        'output_dir', 'extensions', 'saved_model_dir', 'tensorboard_logdir', 'caffe_parser_path'
+    ]
```

Comment:
I think you can create this dictionary automatically. You can extract all argument names from created parser and select ones that have `readable_dirs` type.

Reply:
It will lead to the case from the description, where `data_type` is interpreted as a directory, because we have a folder named `FP32` in the MO root directory

Reply:
I can not extract types, because `get_meta_info` takes as input `argparse.Namespace` which does not contain this information

Reply:
Ok, I got it. Thanks!